user_guide/data_management_tags.rst

```diff
@@ -0,0 +1,152 @@
+.. _user-guide-tags:
+
+Tags
+====
+
+Tag is non-hierarchical keyword or phrase assigned to a record. First of all, tags provide additional information about 
+the records to all the system users. Tags can be assigned to any records of any entity and used for  
+:ref:`filtering <user-guide-filters-management>`, they can be used to create a report only for the records that 
+have/don't have a specific tag and generate targeted segments for mailings and marketing campaigns.
+Moreover, you can easily find all the records with a specific tag, even not knowing 
+their other details from anywhere in the system as described hereunder.  
+
+
+Add a Tag to a Record
+---------------------
+
+You can add a tag from the grid or from a View page of a specific record.
+
+Add a Tag from the Grid
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The tag column is by default available on the :ref:`grids <user-guide-ui-components-grids>`. If they are not, you can 
+:ref:`adjust the table settings <user-guide-ui-components-grids-change-table>` and add the column. Adding a tag is not 
+different from :ref:`editing other properties <user-guide-ui-components-grid-edit>` from a grid:
```

Comment:
Edit: The tag column is available by default on the :ref:`grids <user-guide-ui-components-grids>`. If they are not, you can :ref:`adjust the table settings <user-guide-ui-components-grids-change-table>` and add the column. Adding a tag is no different from :ref:`editing other properties <user-guide-ui-components-grid-edit>` from a grid: